packages/issuer/src/test/Issuer.test.ts

```diff
@@ -66,6 +66,16 @@ describe('Issuer', () => {
         };
     });
 
+    it('gets all certification requests', async () => {
+        const totalVolume = new BigNumber(1e9);
+
+        await createCertificationRequest(totalVolume);
+        await createCertificationRequest(totalVolume);
```

Comment:
Why duplicated?

Reply:
So that it creates two certification requests. And then the tests checks if 2 were created.